x11-misc/polybar/polybar-3.3.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
Is there a reason you offer 3.3.0 and 3.3.1? Say, would anyone choose 3.3.0 over 3.3.1 because they'd deliberately have to mask the latest available version?

Reply:
Just wanted for it to have its consecutive versions. I removed 3.3.0 as it is not really necessary to have.